src/job/job.spec.js

```diff
@@ -25,22 +25,20 @@ describe("job", () => {
       const { email, password } = ADMIN_USER;
       await testWithOtherConnection({ email, password }, async xo => {
         const id = await xo.createJob({
-          job: {
-            ...DEFAULT_JOB,
-            paramsVector: {
-              type: "cross product",
-              items: [
-                {
-                  type: "set",
-                  values: [
-                    {
-                      id: config.vmIdXoTest,
-                      name: "snapshot",
-                    },
-                  ],
-                },
-              ],
-            },
+          ...DEFAULT_JOB,
```

Comment:
The default job should be able to be executed. So, it needs the `paramsVector`. See https://github.com/vatesfr/xo-test/pull/71/files#diff-fdb7684219af8e09af354319f40f8b3dR8